src/App.js

```diff
@@ -83,6 +76,8 @@ const App = () => {
       body: JSON.stringify(newExpense),
     };
 
+    let expenseWasAdded;
```

Comment:
ah, the name tripped me up and I had to check what it means. maybe we can do something like errorOccurred or similar?

Reply:
Sure thing, do you think it makes more sense to initiate it to `false` and then only change the value to `true` if an error is caught? Rather than initiate with no value, I mean.

Reply:
i have no opinion on this